roles/confluent.variables/tasks/main.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Setting HOSTVARs cache for secondary supporting clusters
+  set_fact: 
+    kafka_broker_listeners: "{{ kafka_broker_listeners }}"
+    ssl_enabled: "{{ ssl_enabled }}"
+    pkcs12_enabled: "{{ pkcs12_enabled }}"
+    ssl_mutual_auth_enabled: "{{ ssl_mutual_auth_enabled }}"
+    sasl_protocol: "{{ sasl_protocol }}"
+    kafka_broker_truststore_path: "{{ kafka_broker_truststore_path }}"
+    kafka_broker_truststore_storepass: "{{ kafka_broker_truststore_storepass }}"
+    kafka_broker_keystore_path: "{{ kafka_broker_keystore_path }}"
+    kafka_broker_keystore_storepass: "{{ kafka_broker_keystore_storepass }}"
+    kafka_broker_keystore_keypass: "{{ kafka_broker_keystore_keypass }}"
+    sasl_plain_users: "{{ sasl_plain_users }}"
+    sasl_scram_users: "{{ sasl_scram_users }}"
+    kerberos_kafka_broker_primary: "{{ kerberos_kafka_broker_primary }}"
+    kafka_broker_keytab_path: "{{ kafka_broker_keytab_path }}"
+    kafka_broker_kerberos_principal: "{{ kafka_broker_kerberos_principal  | default('kafka') }}"
+    kafka_broker_ldap_user: "{{ kafka_broker_ldap_user }}"
+    kafka_broker_ldap_password: "{{ kafka_broker_ldap_password }}"
+    mds_bootstrap_server_urls: "{{ mds_bootstrap_server_urls }}"
+    fips_enabled: "{{ fips_enabled }}"
+  delegate_to: "{{ item }}"
+  delegate_facts: true
+  with_items: 
```

Comment:
what happens if the group is not in your inventory file?
might need a `when`

Reply:
The group defaults to `kafka_brokers` which is the normal group. So the user would have to explicitly get the wrong thing here which would be appropriate to fail.

Reply:
actually it is valid to not have kafka_broker in your inventory file. if you do custom properties to configure connect -> ccloud kafka then there will not be a kafka_broker group, lets just put a "when kafka_broker in groups"

Reply:
Changed this to have an `| default(omit)` to handle the no/empty group case instead of using `when`. Should allow for better re-use of this function for other central clusters needing the same cache. eg RBAC.